skim/src/options.rs

```diff
@@ -16,6 +16,22 @@ use crate::{CaseMatching, FuzzyAlgorithm, Selector};
 ///
 /// sk is a general purpose command-line fuzzy finder.
 ///
+/// # MODES
```

Comment:
I'd describe it as:

sk by default reads the to be filtered lines from stdin. sk can also execute a command (specified with --cmd) each time the query is changed to get the results which is called interactive mode. Interactive mode can be used on startup with -i, --interactive or toggled while sk is running with the `toggle-interactive` action, which by default is bound to \<ctrl-q>.

(If there's only two modes I don't think naming the default mode is really necessary and I think it's very important to describe the default behavior.)

Reply:
Oh I just found out that this isn't quite the full picture yet because `-c` can also be used in the normal mode, which is apparently what happens when you run `-c` without `-i`. This should be described in the paragraph I wrote as well.

Reply:
I think what you wrote is too restrictive about all this. I'm going to keep it this way for now, and we'll see if it works in the longer term.